### PR TITLE
Cors middleware

### PR DESCRIPTION
**Pull request by hammacktony**
Adds ability to inject CORS headers into HTTP Response.

The corresponding test is included.

### CODE REVIEW

File: masonite/middleware/CorsMiddleware.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        |--------------------------------------------------------------------------
+        |
+        '''
+
+        # Demo CORS values - Modify for your own CORS values
+        cors = {"Access-Control-Allow-Origin": "*",
+                "Access-Control-Allow-Headers": "Content-Type",
+                "Access-Control-Allow-Methods": "DELETE, GET, HEAD, OPTIONS, PATCH, POST, PUT"}
+
+        for key, value in cors.items():
```
Comment (josephmancuso) [inline masonite/middleware/CorsMiddleware.py]: Theres just something about this I don't like 🤔 

it looks very dirty

Comment (josephmancuso) [reply masonite/middleware/CorsMiddleware.py]: i'm gonna update the headers method real quick to accept a dictionary

Comment (josephmancuso) [reply masonite/middleware/CorsMiddleware.py]: https://github.com/MasoniteFramework/core/pull/339

Comment (josephmancuso) [reply masonite/middleware/CorsMiddleware.py]: ok now we should be able to do something like:

```
self.request.header({"Access-Control-Allow-Origin": "*",
                "Access-Control-Allow-Headers": "Content-Type",
                "Access-Control-Allow-Methods": "DELETE, GET, HEAD, OPTIONS, PATCH, POST, PUT"})
```

a little cleaner

**Comment by hammacktony** [comment]
I am with you on all of those changes. However, I have a slight syntax issue.

```
from masonite.middleware import CorsMiddleware as Middleware


class CorsMiddleware(Middleware):
```
I think the naming is confusing. You have two `CorsMiddleware`. It is not....intuitive I guess. Besides this, I really like your changes.

**Comment by josephmancuso** [comment]
well technically it is 1 middleware. Since you are importing as a different namespace

**Comment by josephmancuso** [comment]
why was this closed again?

**Pull request closed without merging**